src/views/WelcomeView.vala

```diff
@@ -25,8 +25,8 @@ namespace Cipher.Views {
     public class WelcomeView : Gtk.Grid  {
 
         construct {
-            var welcome = new Granite.Widgets.Welcome ("Cipher", "Encode and decode text");
-                welcome.append ("text-x-source", "Contribute more Ciphers ...", "... and sharpen your coding skills.");
+            var welcome = new Granite.Widgets.Welcome (_("Cipher"), _("Encode and decode text"));
+                welcome.append (_("text-x-source"), _("Contribute more Ciphers ..."), _("... and sharpen your coding skills."));
```

Comment:
```suggestion
                welcome.append ("text-x-source", _("Contribute more Ciphers ..."), _("... and sharpen your coding skills."));
```

The first argument is an icon name, so it should not be translatable. See https://valadoc.org/granite/Granite.Widgets.Welcome.append.html